### PR TITLE
MINOR: add clean up command to tests README

### PR DESCRIPTION
**Pull request by gongxuanzhang**
We may need to clean up after running the test

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -47,6 +47,10 @@ TC_PATHS="tests/kafkatest/tests/streams/streams_upgrade_test.py::StreamsUpgradeT
 ```
 bash tests/docker/ducker-ak up -j 'openjdk:11'; tests/docker/run_tests.sh
 ```
+* Clean Docker Image
+```
+docker rm -f $(docker ps -aq --filter "name=ducker")
```
Comment (chia7712) [inline tests/README.md]: Have you considered using `./tests/docker/ducker-ak down -f` instead?

**Review by chia7712 — COMMENTED** [review]
@gongxuanzhang thanks for this patch

File: tests/README.md (hunk 1 above)
Comment (gongxuanzhang) [reply tests/README.md]: you are right,this is even better!

**Comment by gongxuanzhang** [comment]
I update it

File: tests/README.md [hunk 2]
```diff
@@ -47,6 +47,10 @@ TC_PATHS="tests/kafkatest/tests/streams/streams_upgrade_test.py::StreamsUpgradeT
 ```
 bash tests/docker/ducker-ak up -j 'openjdk:11'; tests/docker/run_tests.sh
 ```
+* Clean Docker Image
+```
+./tests/docker/ducker-ak down -f
```
Comment (chia7712) [inline tests/README.md]: Could you please align the style `bash tests/docker/ducker-ak up`?

**Comment by gongxuanzhang** [comment]
@chia7712  thanks for your review,I update it

File: tests/README.md [hunk 3]
```diff
@@ -47,6 +47,10 @@ TC_PATHS="tests/kafkatest/tests/streams/streams_upgrade_test.py::StreamsUpgradeT
 ```
 bash tests/docker/ducker-ak up -j 'openjdk:11'; tests/docker/run_tests.sh
 ```
+* Clean Docker Image
```
Comment (chia7712) [inline tests/README.md]: `remove ducker-ak nodes`

Comment (gongxuanzhang) [reply tests/README.md]: Is "remove ducker-ak images" better?
If you feel the same way, I'll change it to this

Comment (chia7712) [reply tests/README.md]: It does NOT remove any images. The command used to remove images is "prune"

Comment (gongxuanzhang) [reply tests/README.md]: sorry,I was actually going to say Container

Comment (chia7712) [reply tests/README.md]: `Container` is good to me

Comment (gongxuanzhang) [reply tests/README.md]: I update it,thank you

File: tests/README.md [hunk 4]
```diff
@@ -47,6 +47,10 @@ TC_PATHS="tests/kafkatest/tests/streams/streams_upgrade_test.py::StreamsUpgradeT
 ```
 bash tests/docker/ducker-ak up -j 'openjdk:11'; tests/docker/run_tests.sh
 ```
+* Remove ducker-ak Containers
```
Comment (chia7712) [inline tests/README.md]: `containers`

**Comment by gongxuanzhang** [comment]
@chia7712  PTAL

**Review by ardada2468 — APPROVED** [review]

**Pull request merged**